vcx/libvcx/src/credential_def.rs

```diff
@@ -454,15 +436,14 @@ pub mod tests {
     #[cfg(feature = "pool_tests")]
     #[test]
     fn test_create_revocable_cred_def_with_payments() {
-        let wallet_name = "test_create_revocable_cred_def";
-        init!("ledger");
+        let _setup = SetupLibraryWalletPool::init();
 
         let (schema_id, _) = ::utils::libindy::anoncreds::tests::create_and_write_test_schema(::utils::constants::DEFAULT_SCHEMA_ATTRS);
         let did = settings::get_config_value(settings::CONFIG_INSTITUTION_DID).unwrap();
 
-        let revocation_details = json!({"support_revocation": true, "tails_file": get_temp_dir_path(Some("tails.txt")).to_str().unwrap(), "max_creds": 2}).to_string();
+        let revocation_details = json!({"support_revocation": true, "tails_file": get_temp_dir_path("tails.txt").to_str().unwrap(), "max_creds": 2}).to_string();
         let handle = create_and_publish_credentialdef("1".to_string(),
-                                                      wallet_name.to_string(),
+                                                      "test_create_revocable_cred_def".to_string(),
```

Comment:
Again, can we separate the constant from the function call at least?

Reply:
for what?

Reply:
The purpose is not to have inline string literals. It is a bad style of coding.